### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'equals') with Jasmine 4.2

### ISSUE THREAD

**Issue by joonass-visma** [mradionov/karma-jasmine-diff-reporter#63]
When trying to run our tests with Jasmine 4.2, I get the following error:

`TypeError: Cannot read properties of undefined (reading 'equals')`

The error is coming from `util-patch.js`:

```javascript
function matchersUtilPatched(j$) {
  var util = j$.matchersUtil;
  var oldEquals = util.equals.bind(util); // <--- crashes here because util is undefined
...
```
According to [Jasmine 4 upgrade notes](https://jasmine.github.io/tutorials/upgrading_to_Jasmine_4.0) `jasmine.matchersUtil` is deprecated and apparently has been removed now.

**Comment by mradionov** [comment]
Thanks for letting me know, I'll see what I can do.

**Comment by mradionov** [comment]
I've published a new version of the plugin 3.0.0 that will auto-disable the plugin and display a warning in the karma output if it detects that Jasmine 4.x is used. Also added a section at the top of README explaining it.
